src/lib/containers/LoginPage.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react'
+import Login from './Login'
+import SynapseLogoSvg from '../../lib/assets/logos/synapse-logo-blue.svg'
+import { useEffect } from 'react'
+
+export type LoginPageProps = {
+  backgroundElementId?: string  // contain the element id to set the page background if provided
+}
+
+const LoginPage: React.FunctionComponent<LoginPageProps> = (props) => {
+  let mounted = true
+  let parent:HTMLElement|null = null
+  const { backgroundElementId } = props
+  const thisClass = "login-panel-wrapper"
+
+  useEffect(() => {
+    if (mounted) {
+    parent = backgroundElementId ? (document.getElementById(backgroundElementId) as HTMLElement)
+      : (document.getElementById(thisClass)?.parentNode as HTMLElement)
+      if (parent) {
+        parent.classList.add("login-panel-wrapper-bg")
+      }
+    }
```

Comment:
The purpose of this block is to set the background image css class to the parent of this component. The parent could be further up in the DOM tree depending on where is the best place to set the background.

Reply:
I'm unclear why you need to adjust the parent container background, instead of defining this component as the full page, but I agree that this approach should work.

Reply:
I guess I was overthinking this. I wasn't sure how this component will be applied in Synapse so I provided a way to override background in any parent tags just in case. But you brought up a good point so I deleted the code to set background outside this component.

Reply:
No need for useEffect() now

Reply:
Deleted!